app/assets/javascripts/alert.js

```diff
@@ -0,0 +1,25 @@
+(function ($) {
```

Comment:
bao quanh document.ready function đi cho nó tổng quát

Reply:
Em dùng document.ready nó không chạy ạ, toàn bị lỗi function chưa được định nghĩa